appservice/src/utils/ui.ts

```diff
@@ -0,0 +1,26 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export namespace uiUtils {
+
+    export interface PartialList<T> extends Array<T> {
+        nextLink?: string;
+    }
+
+    export async function listAll<T>(client: { listNext(nextPageLink: string): Promise<PartialList<T>>; }, first: Promise<PartialList<T>>): Promise<T[]> {
+        const all: T[] = [];
+
+        for (let list = await first; list.length || list.nextLink; list = list.nextLink ? await client.listNext(list.nextLink) : []) {
+            all.push(...list);
+        }
+
+        return all;
+    }
+
+    export function getSignInCommandString(): string {
```

Comment:
Seems unnecessary to have an exported 'utility' function that returns a constant string and is only used once

Reply:
Now we get to things that I simply copied from the wizard and didn't try to improve.  I'll take a look after the initial push.